docs/integrations/catalog/prefect-spark-on-k8s-operator.yaml

```diff
@@ -0,0 +1,7 @@
+collectionName: prefect-spark-on-k8s-operator
+author: Manoj Babu Katragadda
+authorUrl: https://github.com/tardunge
+documentation: https://tardunge.github.io/prefect-spark-on-k8s-operator/
+iconUrl: /img/collections/kubernetes.png
```

Comment:
Could the logo be a Spark and Kubernetes logo side by side? I think we want to be able to quickly communicate how this is different from `prefect-kubernetes`.

Reply:
yeah. makes sense. i've added one.